Apps/GatewayApi/src/Validations/AssociateDelegationRequestValidator.cs

```diff
@@ -17,21 +17,19 @@
 namespace HealthGateway.GatewayApi.Validations
 {
     using FluentValidation;
-    using HealthGateway.Database.Models;
-    using HealthGateway.GatewayApi.Models;
 
     /// <summary>
-    /// Validates <see cref="Delegation"/> instances.
+    /// Validates associate delegation request.
     /// </summary>
-    public class AssociateDelegationRequestValidator : AbstractValidator<AssociateDelegationRequest>
+    public class AssociateDelegationRequestValidator : AbstractValidator<object>
```

Comment:
Suggest changing the type to `AbstractValidator<string>` and passing in encryptedDelegationId to the Validate() method (instead of as a parameter on the constructor), so the rule would be:
```
this.RuleFor(v => v).NotEmpty().WithMessage("Encrypted Delegation Id must not be empty");
```